### ISSUE TITLE
How many times of exp you have done in paper

### ISSUE THREAD

**Issue by ChiShengChen** [dongyangli-del/EEG_Image_decode#21]
Thanks for your great work, 
I am curious about all your plots in the paper have std range, 
how many times of exp you have done in paper in each plot to get the std bars?

**Comment by dongyangli-del** [comment]
Hi, @ChiShengChen, thanks for your question.

In the bar graphs of our main text (including retrieval and classification tasks), we trained and tested the model with different random initializations 5 times, and the result of each random setup was the maximum test accuracy during the training and testing process.

In the meantime, in the table in our appendix (detailed retrieval tasks), we trained and tested the model with different random initializations 5 times, and the result of each random setup was the average test accuracy of the last 10 epochs during the training and testing process.

Although the experimental setting is fair to different methods, this practice still caused concerns among a few people. Therefore, in the latest version of the revised manuscript, we have reformulated the form of the validation set to achieve a more comprehensive model evaluation.
